manifests/webhook/package.pp

```diff
@@ -1,19 +1,13 @@
 # Private class, do not include it directly.
 # Installs the webhook packages
 class r10k::webhook::package (
-  $is_pe_server = $r10k::params::is_pe_server,
-  $provider     = $r10k::params::provider,
+  $is_pe_server    = $r10k::params::is_pe_server,
+  $provider        = $r10k::params::provider,
+  $sinatra_version = $r10k::params::webhook_sinatra_version,
 ) inherits r10k::params {
   if !defined(Package['sinatra']) {
     package { 'sinatra':
-      ensure   => installed,
-      provider => $provider,
-      before   => Service['webhook'],
-    }
-  }
-  if !defined(Package['rack']) {
-    package { 'rack':
-      ensure   => installed,
+      ensure   => $sinatra_version,
```

Comment:
Ok.  I can't think of any situation where it wouldn't be a gem provider *and* the package would still be called `sinatra` (and not `ruby-sinatra` or something), so a note in the source should be ok?  I'll update it in the morning.